### DISCUSSION TITLE
怎么能让一个组件覆盖在另外一个组件上面？

### DISCUSSION THREAD

**Discussion by wuchangming** [illacloud/illa-builder#671] [Q&A]
如题，目前好像只有容器组件是允许子组件覆盖在父容器上面，但普通组件怎么覆盖在另一个普通组件上面呢？

**Comment by karminski** [comment]
@wuchangming hi, 除了容器类型以外目前组件暂时不支持叠放.

**Reply by wuchangming** [reply]
@karminski 
多问一句哈，是什么原因，最初设计的时候先不支持的呢？是有遇到什么问题，还是说觉得这种使用场景比较少？
